plugins/ca-changesets/bin/ca-changeset-wrapper

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+set -e
+pnpm dlx @changesets/cli "$@"
```

Comment:
This plugin uses pnpm but doesn't specify it. It's relying on _some version_ of pnpm being provided by the system, which isn't great. Suggest using `npx --yes @changesets/cli "$@"`